src/scenes/contact/contactForm.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FormEvent, useRef, useState } from "react";
+import emailjs from "@emailjs/browser";
+import styles from "./contactForm.module.scss";
+
+export function ContactForm() {
+  const [messageSend, setMessageSend] = useState(false);
+  const form = useRef<HTMLFormElement | null>(null);
+
+  const sendEmail = (e: FormEvent) => {
+    e.preventDefault();
+    if (!form.current) return;
+    emailjs
+      .sendForm(
+        `${process.env.REACT_APP_SERVICE_ID}`,
+        `${process.env.REACT_APP_TEMPLATE_ID}`,
+        form.current,
+        `${process.env.REACT_APP_PUBLIC_KEY}`
+      )
+      .then(
+        (result) => {
+          setMessageSend(true);
+          if (form.current && form.current.reset) {
```

Comment:
it this if check required? you have early return in line 11 if the form.current doesn't exists

Reply:
When I delete this if typescript yells me that form.current can be null. So it has to be like that.